2019/capetown/schedule.md

```diff
@@ -0,0 +1,7 @@
+---
+layout: location_detail
+partner_site: bramberg
```

Comment:
This doesn't match the directory it is in. Is this incorrect?

Reply:
addressed in: https://github.com/compsocialscience/summer-institute/pull/1240/commits/45c69738e5def1947158684399c06d8a7735dc0d